Source/Views/TableViews/SPCopyTable.m

```diff
@@ -603,6 +612,10 @@ - (NSString *)rowsAsSqlInsertsOnlySelectedRows:(BOOL)onlySelected skipAutoIncrem
 					case 4:
 						SPLog(@"Not adding auto inc value");
 						break;
+                    // Generated column
```

Comment:
Maybe tweak the comment to be something like "Generated column AND skip generated columns=YES" to be a little more clear that type 5 only exists if skip generated is YES

Reply:
It's done